src/main/java/com/minecolonies/coremod/blocks/BlockHutWareHouse.java

```diff
@@ -1,16 +1,18 @@
 package com.minecolonies.coremod.blocks;
 
 import com.minecolonies.coremod.tileentities.TileEntityWareHouse;
+import net.minecraft.block.Block;
 import net.minecraft.tileentity.TileEntity;
 import net.minecraft.world.World;
+import net.minecraftforge.registries.IForgeRegistry;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'net.minecraftforge.registries.IForgeRegistry'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AUselessImportCheck)